.github/workflows/master.yml

```diff
@@ -0,0 +1,36 @@
+# Push to master workflow.
+#
+# Less strict than the PR workflow because checks have run already.
+
+name: Master
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - master
+
+env:
+  RUST_BACKTRACE: 1
+
+jobs:
+  test:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+      - uses: actions/checkout@v1
+        if: github.event.pull_request.merged
```

Comment:
The conditional can be moved to the Job level as well. It can be placed anywhere before `steps:`
That way we don't need to have the same conditional for every step

[Here's an example](https://github.com/lionel1704/gh-actions-demo/blob/d552e579abf47824756cb39bde96facaa8198c31/.github/workflows/releases.yml#L7)